Bindings/Python/examples/simple-arm.py

```diff
@@ -0,0 +1,135 @@
+# ----------------------------------------------------------------------- #
+# The OpenSim API is a toolkit for musculoskeletal modeling and           #
+# simulation. See http://opensim.stanford.edu and the NOTICE file         #
+# for more information. OpenSim is developed at Stanford University       #
+# and supported by the US National Institutes of Health (U54 GM072970,    #
+# R24 HD065690) and by DARPA through the Warrior Web program.             #
+#                                                                         #
+# Copyright (c) 2005-2012 Stanford University and the Authors             #
+# Author(s): Neil Dhir                                                    #
+#                                                                         #
+# Licensed under the Apache License, Version 2.0 (the "License");         #
+# you may not use this file except in compliance with the License.        #
+# You may obtain a copy of the License at                                 #
+# http://www.apache.org/licenses/LICENSE-2.0.                             #
+#                                                                         #
+# Unless required by applicable law or agreed to in writing, software     #
+# distributed under the License is distributed on an "AS IS" BASIS,       #
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or         #
+# implied. See the License for the specific language governing            #
+# permissions and limitations under the License.                          #
+# ----------------------------------------------------------------------- #
+
+# simple-arm.py
+# Author: Neil Dhir
+# ------------------------------------------------------------------------#
+# ABSTRACT: This short piece of OpenSim python API example mimics         #
+# the simple-arm example found on the core landing page. Though it does   #
+# not include forward simulation, but instead saves the model to an .osim #
+# file which can be used in the GUI for further analysis.		  #
+# ------------------------------------------------------------------------#
```

Comment:
I expect most people will find this file in a distribution of OpenSim--not in the opensim-core source code repository; "core landing page" will not have much meaning in that context. I think this description should explain that the model has two bodies, two joints, a muscle, and a controller.

You could also consider giving the file a name that indicates it is a model-building example (`build-simple-arm-model.py`?).